TombLib/TombLib/Catalogs/TEN Node Catalogs/Timers.lua

```diff
@@ -0,0 +1,274 @@
+local Timer = require("Engine.Timer")
+LevelFuncs.noA = function() end
```

Comment:
What is this? If you have to pass null function, you can try to pass `nil` instead.

Reply:
![image](https://github.com/MontyTRC89/Tomb-Editor/assets/116632612/d8b4a734-2a31-4209-a6bd-7a268d938afa)
![image](https://github.com/MontyTRC89/Tomb-Editor/assets/116632612/3a549ed7-4ee2-4851-861b-06364e1917c0)
If I try to pass a null value there is an error

Reply:
You need to fix underlying code then. It seems @hispidence forgot to add a nil check to a callback function.

Please go to `Scripts\Engine\Timer.lua` file and go to line 122. You will find this block of code:

```
				if t.remainingTime <= 0 then
					if not t.loop then
						t.active = false
					else
						t.remainingTime = t.remainingTime + t.totalTime
					end
					t.func(table.unpack(t.funcArgs))
				end
```

Change it to:

```

				if t.remainingTime <= 0 then
					if not t.loop then
						t.active = false
					else
						t.remainingTime = t.remainingTime + t.totalTime
					end
					
					if (t.func ~= nil) then
						t.func(table.unpack(t.funcArgs))
					end
				end
```

This should fix your issue. If it does, I will push this change to TEN repo.

Reply:
with this edit the nil value is accepted and of course the timer does not execute any function when finished. The other timers with an assigned function work perfectly!